topiary-cli/Cargo.toml

```diff
@@ -74,11 +74,12 @@ default = [
 contributed = [
   "css",
   "openscad",
+  "sdml",
 ]
 
 # Excluded by default
 experimental = [
-  "rust"
+  "rust",
```

Comment:
Nitpick: The comma is now redundant. If you remove it, it will make the diff cleaner.